Docs: Use OpenJS Foundation logo in header

kborchers opened the pull request:
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/webhint.io)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

## Short description of the change(s)

Replaces the JS Foundation logo and link in the header with the OpenJS Foundation logo and link.

sarvaje commented on src/webhint-theme/source/images/openjsf.svg:
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Can you use some tools like: https://jakearchibald.github.io/svgomg/ to reduce the size of the logo?

sarvaje replied:
I mean the size in bytes not in px hehehe.

antross commented:
@sarvaje Is there an easy way we can spin up a staging version of the site with this change to see how it looks in context?

sarvaje commented:
I don't know, but I can run it on my machine xD

antross commented:
> I don't know, but I can run it on my machine xD

That works if you want to just take a screenshot and add it here. :)

sarvaje commented:
![image](https://user-images.githubusercontent.com/1581288/61752255-e19a6080-ad5f-11e9-8c03-96b04ab5df54.png)

IMO, it look a little bit weird, but I'm not a designer hehehe

ststimac commented:
Yeah that definitely looks weird because there’s stacked lettering, it’s not just a single line of text with an icon. I suggest we do the following: 

- rather than include the logo in the small banner at the top, just have “An Open JS Foundation Project” as text and make “Open JS Foundation” a link. 

- Include the Open JS Foundation logo elsewhere on the page. Maybe the footer?

kborchers commented:
I agree with @ststimac and as long as the logo is on the site, I don't think the foundation is too worried about where it lands as long as it's there and the project is happy with it. I will push changes shortly to address that request as well as remove the old logo and try shrinking the size of the svg file.

antross commented:
Sounds good. 

Thanks @kborchers for opening the PR and taking care of the updates. We really appreciate it! 😊

kborchers commented:
I have added updates including:
- moved the logo to the footer
- positioned it under the social links but feel free to suggest a different position or positioning css if there is a better way as I am no expert with css
- shrunk the svg file and included it inline in the footer to allow for text color changes with different color footer backgrounds
- deleted both the old logo as well as the separate svg file for the new logo since it is inline now

sarvaje commented:
This is how it looks now.

![image](https://user-images.githubusercontent.com/1581288/61904981-d3be1a00-aedc-11e9-8d62-952ec8b17fde.png)

sarvaje commented:
@kborchers thank you very much for the PR.

I'm going to push some changes to fix the responsiveness of the site. I also change some class names to fit a little bit better with the styles in the site.

Thank you very much!!

antross commented:
@sarvaje Any final updates you want to make or can I merge this?

sarvaje commented:
@antross you can merge it :)
